### PR TITLE
Performance improvements

### PR DESCRIPTION
**Pull request by MaxMartiFoundry**
- Resizes some client logos that were very large
- Take advantage of the squarespace image loader where possible
  - Even for images that aren't hosted on ss static, it defers load to DOM Ready so they get out of the way of critical assets
- Refactor team photos script to use the image loader

Changes the package scripts to make it easier to run the server.

### CODE REVIEW

**Comment by ghost** [comment]
@MaxMartiFoundry consider adding a transition behavior to the images when they load? What's the effect look like when they get loaded via the `data-src` attribute?

File: home.region [hunk 1]
```diff
@@ -66,38 +66,39 @@
       </div>
 
       {#  Client logos carousel  }
+      {#  Uses the data-src attribute to take advantage of the squarespace loader's deferred load - but does not actually perform resize because they are "local" assets }
```
Comment (panoptican) [inline home.region:69]: Neat!

**Review by panoptican — APPROVED** [review]
Nice to have some real coders coding in here!

**Comment by MaxMartiFoundry** [comment]
> @MaxMartiFoundry consider adding a transition behavior to the images when they load? What's the effect look like when they get loaded via the `data-src` attribute?

The images use the fade in/slide up animation that seems to be the default.

**Review by nikmanchan — APPROVED** [review]
LGTM

**Pull request merged**